### PR TITLE
#13834 Standardize salesforce-endpoint

### PR DESCRIPTION
**Pull request by pasaperez-work**
Pull request for the topic https://github.com/slingr-stack/platform/issues/13834 created by @pasaperez-slingr 

## What does it do?

This ticket implements the use of slingr-helpgen, whereby it modifies existing functions, adds basic functions and a flow step.

## How to test it?

Make calls from the platform for example using the console using the information provided in the documentation.

## Implementation notes

Create a new tags changing the major version of the endpoint, it's partially backward compatible.

## Implementation notes

N/A

### CODE REVIEW

**Comment by pasaperez-slingr** [comment]
In draft process: still needs 2 or 3 business steps and integration testing.

**Comment by sebasmoyano** [comment]
#approve

File: scripts/helpersDocs.md [hunk 1]
```diff
@@ -0,0 +1,1050 @@
...
+
+
+### Custom Flow Steps Name
+
+Description of Custom Flow Steps
+
+*MANUALLY ADD THE DOCUMENTATION OF THESE FLOW STEPS HERE...*
+
+
+</details>
```
Comment (agreggio-slingr) [inline scripts/helpersDocs.md:1047]: Add custom flow step documentation

File: scripts/http-generic-helpers.js
Comment (agreggio-slingr) [inline scripts/http-generic-helpers.js:1]: Wouldn't it be better to leave this js file for backwards compatibility?

File: scripts/salesforce-helpers.js
Comment (agreggio-slingr) [inline scripts/salesforce-helpers.js:1]: Wouldn't it be better to leave this js file for backwards compatibility?

File: scripts/helpersDocs.md (hunk 1 above)
Comment (pasaperez-work) [reply scripts/helpersDocs.md:1047]: The documentation of the customs steps is in the Readme file, it just remains as a generated file.

File: scripts/http-generic-helpers.js
Comment (pasaperez-work) [reply scripts/http-generic-helpers.js:1]: These helpers are exactly the same as the `functions.js` file.

File: scripts/salesforce-helpers.js
Comment (pasaperez-work) [reply scripts/salesforce-helpers.js:1]: These helpers are not so old, they respect our current path+method naming standard. It is one of the first versions used with the helper generator (see the `gen/generatorConfig.json` file which was renamed to `gen/helperGen.json` and slightly edited to follow the standard, but the output is similar in both cases, fixing bugs in the new generation).

**Pull request merged**